server/config/queries.js

```diff
@@ -1,19 +1,27 @@
 const createUser = `
-        INSERT INTO users (
-                firstName,
-                lastName,
-                employmentDate,
-                membershipDate,
-                nationality,
-                phoneNumber,
-                email,
-                profileImage,
-                slackHandle,
-                salt,
-                hash
-            ) VALUES ($1,$2,$3,$4,$5,$6,$7,$8,$9,$10,$11)
+    INSERT INTO users (
```

Comment:
Data for inserting the new user are different from table columns of creating user table, check that

Reply:
Which query is that? because i added a new query for updating user info and another for creating a user.

Reply:
Now I understand the logic u put there.